src/TRoomDB.h

```diff
@@ -80,8 +80,14 @@ class TRoomDB
     void restoreSingleRoom(int, TRoom*);
     const QString getDefaultAreaName() { return mDefaultAreaName; }
 
-
-    QMap<QString, int> hashTable;
+    // This is for muds that provide hashes to rooms instead of IDs.
+    // If it exists, we delete the info when deleting a room.
+    // But we rely on the user to add the data, don't do any checking,
+    // and it isn't audited.
+    // It should be a QHash, but changing it would break loading from files
```

Comment:
I think it would still make sense to only store one direction in the file/ioStream. That way there's no chance that they are out of sync at load time, you have a smaller file/ioStream, and depending on the stream source (nework|HDD|SSD|memoryMap) iterating over one collection to build another may be faster than reading from a stream to build it.

Still, if the file version is going to be changed, I can update it pretty easily. I'm pretty sure all methods we use map directly from `QMap` to `QHash`. Just need to change the types.